### PR TITLE
mail-client/alpine: Support USE=pam

### PR DESCRIPTION
**Pull request by vaeth**

### CODE REVIEW

**Comment by nvinson** [comment]
@hattya , @mschiff , @radhermit , or @robbat2 (net-mail project)

**Comment by monsieurp** [comment]
ping?

File: mail-client/alpine/alpine-2.00-r7.ebuild [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	ebegin "Unbundling the c-client library"
+	rm -rf "${S}"/imap
+	local f
+	while read f ; do
+	sed -i -e \
+		's~^#include[[:blank:]]".*/c-client/\(.*\)"~#include <imap/\1>~g' "$f"
+	done < <(find "${S}" -name "*.c" -o -name "*.h")
+	eend $?
+
+	! use pam || eapply "${FILESDIR}"/2.00-lpam.patch
```
Comment (dwfreed) [inline mail-client/alpine/alpine-2.00-r7.ebuild]: wat

Comment (monsieurp) [inline mail-client/alpine/alpine-2.00-r7.ebuild]: @vaeth hey. Could you please do something a bit wordier yet easier to read for all?
```if ! use pam; then
  epatch ...
fi```

**Comment by vaeth** [comment]
I have followed the request, but have perhaps done a stupid thing:
In order to not generate a redundant commit with only that trivial change (which nobody needs in the git history later on), I had done a ```git commit --amend && git push --force-with-lease```, not being aware that this will remove your comments which were apparently bound by github to the previous commit hash.
Not sure exactly what to do now...

**Comment by mgorny** [comment]
I don't think this package is going to work through PRs. Please file a Gentoo bug instead.

**Comment by vaeth** [comment]
https://bugs.gentoo.org/show_bug.cgi?id=617924

**Pull request closed without merging**